lib/cli.ex

```diff
@@ -5,10 +5,10 @@ defmodule Onigumo.CLI do
 
   def main(argv) do
     try do
-      {[], [component]} = OptionParser.parse!(argv, strict: [])
+      {[], [component], []} = OptionParser.parse(argv, strict: [])
```

Comment:
Just mentioning that in this case, there are now two different MatchErrors that can occur:

- An invalid switch is provided, causing the third assignment `[]` to not match.
- More than one positional argument causing the secund assignment `[component]` to fail.

We don’t, however, need to differentiate these two, as both mean invalid CLI usage.